python/ray/data/_internal/execution/operators/actor_pool_submitter.py

```diff
@@ -28,8 +28,6 @@ def __init__(
             ray_remote_args: Remote arguments for the Ray actors to be created.
             pool_size: The size of the actor pool.
         """
-        if "num_cpus" not in ray_remote_args:
```

Comment:
@ericl I reverted this, since it looks not compatible with the requirement that "num_gpus and num_cpus cannot be both specified" (user may want to run actors on gpus and then they won't be able tp specify cpus).